modules/plugin/jdbc/jdbc-postgis/src/test/java/org/geotools/data/postgis/PostGISHStoreTestSetup.java

```diff
@@ -0,0 +1,51 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2017, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.data.postgis;
+
+import org.geotools.jdbc.JDBCDelegatingTestSetup;
+
+public class PostGISHStoreTestSetup extends JDBCDelegatingTestSetup {
+
+    protected PostGISHStoreTestSetup() {
+        super(new PostGISTestSetup());
+    }
+
+    @Override
+    protected void setUpData() throws Exception {
+        dropTestHStoreTable();
+        createTestHStoreTable();
+    }
+
+    private void createTestHStoreTable() throws Exception {
+        String sql = "CREATE TABLE \"hstoretest\" ("
+                + "\"id\" INT, \"name\" VARCHAR, \"mapping\" HSTORE, PRIMARY KEY(id))";
```

Comment:
I believe one has to run "create extension hstore if not exists" in the database to have hstore run correctly, I'd add that to the setup commands (and it should be there pretty much always, it's an old one and afaik it's built-in but not enabled by default).

Reply:
Right. I think I did that on PGAdmin to play with hstore before doing any coding.
Going to add that.